packages/react-native/ReactAndroid/src/main/java/com/facebook/react/HeadlessJsTaskService.java

```diff
@@ -179,31 +179,30 @@ protected ReactContext getReactContext() {
   }
 
   private void createReactContextAndScheduleTask(final HeadlessJsTaskConfig taskConfig) {
-    final ReactHost reactHost = getReactHost();
-
-    if (reactHost == null) { // old arch
-      final ReactInstanceManager reactInstanceManager =
-          getReactNativeHost().getReactInstanceManager();
-
-      reactInstanceManager.addReactInstanceEventListener(
+    if (ReactNativeFeatureFlags.enableBridgelessArchitecture()) {
```

Comment:
Tested this locally, worked with this slight tweak:

```suggestion
    if (ReactFeatureFlags.enableBridgelessArchitecture) {
```

Reply:
Yeah that's because we replaced `ReactFeatureFlags` with `ReactNativeFeatureFlags` on `main`, but on the release branch we still reference the old class

Reply:
Ah of course, and my reproducer was 0.76.2 still (not even .3)
I was confused how that even compiled otherwise and it was sailing right through CI! Makes sense

Okay, then given this was really just symbol motion, not really a semantic change, you've nailed it.